app/assets/javascripts/discourse/app/form-kit/components/fk/control-wrapper.gjs

```diff
@@ -52,42 +49,34 @@ export default class FKControlWrapper extends Component {
       data-disabled={{@field.disabled}}
       data-name={{@field.name}}
       data-control-type={{this.controlType}}
+      {{didInsert (fn @registerField @field.name @field)}}
       {{willDestroy (fn @unregisterField @field.name)}}
     >
-      {{#if @field.showTitle}}
-        <FKLabel
-          class={{concatClass
-            "form-kit__container-title"
-            (if this.titleFormat (concat "--" this.titleFormat))
-          }}
-          @fieldId={{@field.id}}
-        >
-          <span>{{@field.title}}</span>
-
-          {{#unless @field.required}}
-            <span class="form-kit__container-optional">({{i18n
-                "form_kit.optional"
-              }})</span>
-          {{/unless}}
+      {{#unless (eq @field.type "checkbox")}}
```

Comment:
This is the special case introduced. If the component is a checkbox we don't render the title/description/tooltip/optional hint on top of it, this will now be rendered inline. If you need a general title for a group of checkboxes, use checkboxgroup